components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5321,95 +5324,216 @@ def generate_imports(config, cgthings, descriptors, callbacks=None, dictionaries
 
     return CGImports(cgthings, descriptors, callbacks, dictionaries, enums, [
         'js',
-        'js::{JS_CALLEE, JSCLASS_GLOBAL_SLOT_COUNT}',
-        'js::{JSCLASS_IS_DOMJSCLASS, JSCLASS_IS_GLOBAL, JSCLASS_RESERVED_SLOTS_MASK}',
+        'js::JS_CALLEE',
```

Comment:
`_` is greater than `C`, so this should be put after the `js::JSCLASS_*` declarations.